image/controllers.py

```diff
@@ -20,24 +22,18 @@
 HTTP_OK = 200
 TWITTER = "twitter"
 FACEBOOK = "facebook"
+MAPLIGHT = "maplight"
+VOTE_SMART = "vote_smart"
+MAPLIGHT_URL_NOT_FOUND = "maplight url not found"
+VOTE_SMART_URL_NOT_FOUND = "votesmart url not found"
 FACEBOOK_USER_DOES_NOT_EXIST = "facebook user does not exist"
```

Comment:
Whenever we create constants for storage in the database, I would like to have the text (Ex/ "MAPLIGHT_URL_NOT_FOUND") match the python constant, including underscores. So "maplight url not found" should be converted to "MAPLIGHT_URL_NOT_FOUND")